lib/matplotlib/collections.py

```diff
@@ -193,9 +192,7 @@ def __init__(self,
         else:
             self._joinstyle = None
 
-        # default to zeros
-        self._offsets = np.zeros((1, 2))
-
+        self._offsets = offsets
```

Comment:
Could one move this to after the if-clause? (And more the identical line out of it.)

Reply:
Yep, good call, updated.